t/core/profile.t

```diff
@@ -32,3 +32,19 @@ __DATA__
 --- request
 GET /t
 --- error_code: 404
+
+=== TEST 2: set env "APISIX_PROFILE" to Empty String
+--- config
+    location /t {
+        content_by_lua_block {
+            local profile = require("apisix.core.profile")
+            profile.apisix_home = "./test/"
+            profile.profile = ""
```

Comment:
We can check if `profile.profile` is equal to `""` here?

Reply:
`profile.profile = ""` means we set env "APISIX_PROFILE" to empty string.
This is just to simulate a scenario where the environment variable is empty string.

Reply:
> We can check if `profile.profile` is equal to `""` here?

profile.profile = "" means we set env "APISIX_PROFILE" to empty string.
This is just to simulate a scenario where the environment variable is empty string.